src/lib/form-field/form-field.scss

```diff
@@ -108,6 +108,9 @@ $mat-form-field-default-infix-width: 180px !default;
   // Hide the label initially, and only show it when it's floating or the control is empty.
   display: none;
 
+  // Helps prevent text jumpiness when transforming the label.
```

Comment:
Can you provide any background on _why_ this is so?

Reply:
Changed some things so this is no longer needed